server/routes/positions.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const express = require('express');
+const logger = require('../util/logger');
+
+const router = express.Router();
+const Position = require('../models/Position');
+
+router.get('/search', async (_req, res) => {
+  try {
+    const searchFilter = _req.query.filter.replace(/[^a-z\s-']/gi, '');
```

Comment:
Can you move the regex into a separate const variable that explains what it is doing?